python/tests/spark_test_case.py

```diff
@@ -0,0 +1,72 @@
+#
+#  Copyright 2017 TWO SIGMA OPEN SOURCE, LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+'''
+    The common code for all Flint unit tests
+'''
+import os
+import sys
+import collections
+from base_testcase import BaseTestcase
```

Comment:
As this is run from the `flint/python` directory the correct path is `tests.base_test_case`. I've now qualified all the local includes that way so they don't collide.